CHANGELOG.rst

```diff
@@ -8,6 +8,11 @@ Release Notes
 Prelude
 -------
 
+Please note that a critical bug has been identified in this release affecting the kubernetes integration.
```

Comment:
this is a bit too vague and worrying IMO :)

let's be a bit more precise here, something like "the `KUBERNETES` environment variable would not enable the collection of kubelet metrics" maybe?